diff --git a/targets/ARCH/BLADERF/USERSPACE/LIB/bladerf_lib.c b/targets/ARCH/BLADERF/USERSPACE/LIB/bladerf_lib.c
index 3ced5ca83..5e5490562 100644
--- a/targets/ARCH/BLADERF/USERSPACE/LIB/bladerf_lib.c
+++ b/targets/ARCH/BLADERF/USERSPACE/LIB/bladerf_lib.c
@@ -999,10 +999,10 @@ int device_init(openair0_device *device, openair0_config_t *openair0_cfg) {
     // Example of CLI output: RX Frequency: 2539999999Hz
 
 
-    if ((status=bladerf_set_gain_mode(brf->dev, BLADERF_MODULE_RX, BLADERF_GAIN_MGC))) {
+/*    if ((status=bladerf_set_gain_mode(brf->dev, BLADERF_MODULE_RX, BLADERF_GAIN_MANUAL))) {
         fprintf(stderr, "[BRF] Failed to disable AGC\n");
         brf_error(status);
-    }
+    }*/
 
     if ((status=bladerf_set_frequency(brf->dev, BLADERF_MODULE_RX, (unsigned int) openair0_cfg->rx_freq[0])) != 0) {
         fprintf(stderr,"Failed to set RX frequency: %s\n",bladerf_strerror(status));
